stable/vpa/templates/tests/create-vpa.yaml

```diff
@@ -13,6 +13,8 @@ spec:
   serviceAccountName: {{ include "vpa.fullname" . }}-test
   containers:
     - name: test
+      securityContext:
+        {{- toYaml .Values.tests.securityContext | nindent 8 }}
```

Comment:
I would check the nindent value here. Quick guess is that it should be 6, but I'm not sure. (applies to other files too)